packages/beacon-node/src/node/nodejs.ts

```diff
@@ -161,7 +161,7 @@ export class BeaconNode {
     metricsRegistries = [],
   }: BeaconNodeInitModules): Promise<T> {
     if (hasher.name !== "hashtree") {
-      throw Error(`Loaded incorrect hasher ${hasher.name}, expected hashtree`);
+      logger.warn("hashtree is not supported, using hasher", {hasher: hasher.name});
```

Comment:
we added this console log here https://github.com/ChainSafe/lodestar/blob/b819263288b53498a4487ef1307b532a73178cb1/packages/cli/src/applyPreset.ts#L18-L20

I am really not a fan of using console log, do we wanna remove it? (it as more details though, not sure if the user needs to know those though)

Reply:
if we're already catching the case in src/node/nodejs.ts I'd say the console log can be removed